xla/service/cpu/ir_emitter.cc

```diff
@@ -1289,6 +1289,10 @@ Status IrEmitter::HandleAllReduce(HloInstruction* crs) {
   return HandleAllReduceMultipleReplica(crs);
 }
 
+Status IrEmitter::HandleReduceScatter(HloInstruction* rs) {
+  return Unimplemented("ReduceScatter is not implemented on CPU.");
```

Comment:
@jurahul, @burmako, @radhakrishnaba I think this is correct. Let me know if you want me to remove this.

Reply:
This seems fine to me,